src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -1111,6 +1140,52 @@ void FancyZones::OnEditorExitEvent() noexcept
     JSONHelpers::FancyZonesDataInstance().SaveFancyZonesData();
 }
 
+std::vector<HMONITOR> FancyZones::GetMonitorsSorted()
+{
+    std::shared_lock readLock(m_lock);
+
+    std::vector<std::pair<HMONITOR, RECT>> monitorInfo;
+    for (const auto& [monitor, window] : m_zoneWindowMap)
+    {
+        if (window->ActiveZoneSet() != nullptr)
+        {
+            MONITORINFOEX mi;
+            mi.cbSize = sizeof(mi);
+            GetMonitorInfo(monitor, &mi);
+            monitorInfo.push_back({ monitor, mi.rcMonitor });
+        }
+    }
+
+    auto compare = [](const auto& a, const auto& b) {

```

Comment:
autos reduces clearity of intentins here. 
Also usualy this parameters are named 
a -> lhs,
b -> rhs
Which stands for left-right hand side

Reply:
Compare -> isGreater